.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Team
+* @manh-t
```

Comment:
Could you add Rajib and me to the CODEOWNERS file?

Reply:
@nvminhtue added in [b68ff78](https://github.com/manh-t/react-survey/pull/39/commits/b68ff7877896343f127df50578d91e3d4360f9f1)